manager/docs/docs.go

```diff
@@ -0,0 +1,61 @@
+// Package docs GENERATED BY THE COMMAND ABOVE; DO NOT EDIT
+// This file was generated by swaggo/swag
+package docs
+
+import "github.com/swaggo/swag"
+
+const docTemplate = `{
+    "schemes": {{ marshal .Schemes }},
+    "swagger": "2.0",
```

Comment:
`github.com/swaggo/swag` supports only OpenAPI 2.0, while we're using OpenAPI 3.0 frameworks in other projects for years already. Not sure if it's a problem.

Reply:
Yes I have noticed that and tried to get inspired from the patchman app, now in the script generate script the 2.0 openapi spec gets transformed to the 3.0 through swagger online converter. The thing is that the generation script is more complicated. I could not find any golang module for generation of 3.0 API. If you have better idea how to do this or wanna write the spec file by ourselves, I am open to new ideas.